File: c-plus-plus/URI_1012.cpp
#include <iostream>
#include <cmath>
#include <iomanip>
#define pi 3.14159
using namespace std;
int main ()
{
	double a, b, c;
	cin >> a >> b >> c;
	double triangulo, circulo, trapezio, quadrado, retangulo;
	triangulo = (a * c) / 2.0;
	circulo = pi * (pow(c, 2));
	trapezio = ((a + b) * c) / 2.0;
	quadrado = pow (b, 2);
	retangulo = a * b;
	cout << "TRIANGULO: " << fixed << setprecision(3) << triangulo << endl;
	cout << "CIRCULO: " << fixed << setprecision(3) << circulo << endl;
	cout << "TRAPEZIO: " << fixed << setprecision(3) << trapezio << endl;
	cout << "QUADRADO: " << fixed << setprecision(3) << quadrado << endl;
	cout << "RETANGULO: " << fixed << setprecision(3) << retangulo << endl;
	return 0;
}

File: ruby/URI_1012.rb
valores = gets.split
a = valores[0].to_f
b = valores[1].to_f
c = valores[2].to_f

triangulo = a * c / 2.0
circulo = c ** 2 * 3.14159
trapezio = (a + b) * c / 2.0
quadrado = b ** 2
retangulo = a * b

puts "TRIANGULO: %.3f" % triangulo
puts "CIRCULO: %.3f" % circulo
puts "TRAPEZIO: %.3f" % trapezio
puts "QUADRADO: %.3f" % quadrado
puts "RETANGULO: %.3f" % retangulo